Dockerfile

```diff
@@ -11,24 +11,19 @@ RUN yarn --frozen-lockfile && \
 
 # remove development dependencies
 RUN yarn install --production --ignore-scripts --prefer-offline
-RUN yarn cache clean
 
-FROM node:12.18-alpine
+RUN echo "{\"commitTag\": \"${COMMIT_TAG}\"}" > committag.json && \
+  rm -rf src && \
+  rm -rf server
```

Comment:
I don't like the idea of combining two entirely different operations into a single step.

Reply:
Split into two `RUN` statements now!